### DISCUSSION TITLE
iOS 14 panel 3 Column UISplitViewController

### DISCUSSION THREAD

**Discussion by Hackmodford** [MvvmCross/MvvmCross#4041] [Ideas]
iOS 14 introduced a new three column layout for the SplitView https://developer.apple.com/documentation/uikit/uisplitviewcontroller

Any thoughts on how the ViewPresenter could be updated to support this? Is it just a matter of extending the MvxSplitViewPresentationAttribute and MvxIosViewPresenter?

**Comment by Cheesebaron** [comment]
I think it would be as "simple" as extending the `MvxSplitViewPresentationAttribute` and `MvxIosPresenter` to support this. I haven't dug more into how it works though. However, sounds to me it is just another area where you can present details.
